docs/docs/platform/multiple-environments.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Multiple Environments'
```

Comment:
This new guide is clear, but they could still be confused by `development`/`production` variables in the dashboard settings.

To set different env vars in their `staging` application, they would end up setting `production` environment variables

Reply:
True. I think we'll have to live with it for now, but we need to address it soon.